inject/chrome.js

```diff
@@ -20,7 +20,28 @@ function randomString () {
     return num.toString().replace('0.', '')
 }
 
+/**
+ * @see ../src/trackers.js
+ */
+function isTrackerOrigin (trackerLookup) {
```

Comment:
@jonathanKingston I've just copied and pasted this in instead of importing as it seems the chrome inject.js doesn't go through rollup. Would that be an easy change, that I could tackle in this PR, or would I be best leaving it as-is?

Reply:
I think the change would be made here: https://github.com/duckduckgo/content-scope-scripts/blob/main/scripts/inject.mjs#L77 I don't think there's any rationale not to do that other than we didn't need it initially. :)